pkgs/by-name/os/osmo-rtl-sdr/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, cmake
+, pkg-config
+, libusb1
+}:
+
+stdenv.mkDerivation rec {
+  pname = "osmo-rtl-sdr";
```

I think making the variant a suffix is maybe better?
```suggestion
  pname = "rtl-sdr-osmocom";
```

Please also move it then to the corresponding by-name path.